container.go

```diff
@@ -85,7 +87,7 @@ type ImageBuildInfo interface {
 // rather than using a pre-built one
 type FromDockerfile struct {
 	Context        string                         // the path to the context of the docker build
-	ContextArchive io.Reader                      // the tar archive file to send to docker that contains the build context
+	ContextArchive io.ReadSeeker                  // the tar archive file to send to docker that contains the build context
```

Comment:
I guess we are using a Seeker here for allowing more performant retries [here](https://github.com/testcontainers/testcontainers-go/pull/2727/files#diff-14bacfb63a209323729af295c64c9bb978c84e810d18e3e7b7ae66d9b3cd7acaR357), right? Let's be clear in the release notes about the motivation for this change. Could you write a meaningful and short proposal for it 🙏 ?

Reply:
Not more performance, we need to be able to rewind to do multiple passes.